### PR TITLE
postgresql@9.4 9.4.20

### PR DESCRIPTION
**Pull request by matthieuprat**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

This changeset bumps PostgreSQL from 9.4.19 to 9.4.20.

PG 9.4.20 now seems to have [proper support](https://github.com/postgres/postgres/commit/a5361b5933f7b0e0349c34ce11cc7a9438a04c23) for Mojave so we can revert at the same time the changes introduced by ad06b1a04e.

### CODE REVIEW

File: Formula/postgresql@9.4.rb [hunk 1]
```diff
@@ -49,19 +50,6 @@ def install
       args << "--with-tclconfig=#{MacOS.sdk_path}/System/Library/Frameworks/Tcl.framework"
     end
 
-    # As of Xcode/CLT 10.x the Perl headers were moved from /System
```
Comment (javian) [inline Formula/postgresql@9.4.rb]: Can you explain why this change was made and perl was added as a dep?

Comment (matthieuprat) [reply Formula/postgresql@9.4.rb]: Sorry, this PR was a bit blunt! I ended up removing the `perl` dep. After some digging, I figured the actual fix was to define `PG_SYSROOT`:

https://github.com/Homebrew/homebrew-core/blob/a10973b2ef03410ada4f94cc95a7f291cdbb30b6/Formula/postgresql%409.4.rb#L27

I updated the PR's description and the commit message with a short explanation.

**Comment by fxcoudert** [comment]
Thanks @matthieuprat, we love PRs… especially the ones that make our formulas simpler :)
